pytorch_lightning/plugins/precision/precision_plugin.py

```diff
@@ -259,3 +259,15 @@ def load_state_dict(self, state_dict: Dict[str, Any]) -> None:
             state_dict: the precision plugin state returned by ``state_dict``.
         """
         pass
+
+    def on_save_checkpoint(self, checkpoint: Dict[str, Any]) -> None:
```

Comment:
I would also be fine dropping it completely

Reply:
I also don't have a strong preference on keeping this or not